docs/libraries/how-to-run-the-website-in-your-local-server.md

```diff
@@ -19,14 +19,9 @@ It can be useful to know an alternative in order to avoid running all the steps
 ```sh
 git clone https://github.com/arrow-kt/arrow-site.git
 cd arrow-site
-perl -pe "s/\/docs//g" -i docs/_data/features.yml
-./gradlew runAnk
+rm docs/index.md
```

Comment:
Why is this necessary as preparation for the website? 🤔
If I understood correctly from this update, I only need to overwrite the new changes I want to test in the `arrow-site` dir.

Reply:
Right! It's removed because maybe you don't have all the landscape snippets (core, fx, optics) and it won't work when generating the index. However, they are the manual steps and the recommended way is via Gradle task: https://github.com/arrow-kt/arrow/blob/re-org-site/docs/libraries/how-to-run-the-website-in-your-local-server.md#via-gradle-task-recommended

```
./gradlew runArrowSite
```